app/Http/Controllers/BeatmapsController.php

```diff
@@ -36,7 +36,7 @@ public function __construct()
     private static function assertSupporterOnlyOptions(?User $currentUser, string $type, array $mods): void
     {
         $isSupporter = $currentUser !== null && $currentUser->isSupporter();
-        if ($type !== 'global' && !$isSupporter) {
+        if (!in_array($type, ScoreSearchParams::FREE_TYPES, true) && !$isSupporter) {
```

Comment:
wonder if it makes more sense to make it `SUPPORTER_TYPES` now if the condition is no longer simply `!== 'global'`